codex-ui/src/vue/components/card/Card.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div :class="[$style.card, orientation === 'horizontal' ? $style['card--horizontal'] : '']">
+    <div :class="$style['card__cover']"></div>
+
+    <div :class="$style['card__body']">
+      <Heading :level="3">{{ title }}</Heading>
+
+      <div :class="[$style['card__subtitle'], 'text-ui-subtle']">{{ subtitle }}</div>
+    </div>
+  </div>
```

Comment:
where is the slot?

Reply:
I do not see why it's needed. Will add on demand

Reply:
It is needed in horisontal card view in Marketplace


![image](https://github.com/codex-team/notes.web/assets/3684889/98fe6356-1c24-4dee-b7b0-d7504d1361b2)

Reply:
returned it back